Core/Object Arts/Dolphin/IDE/Base/CompilerTest.cls

```diff
@@ -1086,6 +1086,18 @@ testSpecialLiteralsAsUnarySelectors
 				raise: self compilationErrorClass.
 			self denyIsNil: method]!
 
+testStringDoesNotReferToSymbol
```

Comment:
A test for #refersToLiteral: between symbols and strings does not belong here as it is nothing to do with the compiler. You don't need to involve a method at all - the logic of searching through the literals is not relevant to what you are trying to test. I would just keep it simple and put it in StringTest/SymbolTest invoking String>>refersToLiteral: and Symbol>>refersToLiteral: directly in the necessary combinations.

Reply:
Yeah, good point, done.